examples/orthographic.js

```diff
@@ -10,34 +10,20 @@ var extent = new itowns.Extent(
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
 
-var r = viewerDiv.clientWidth / viewerDiv.clientHeight;
-var camera = new itowns.THREE.OrthographicCamera(
-    extent.west(), extent.east(),
-    extent.east() / r, extent.west() / r,
-    0, 1000);
-
 // Instanciate PlanarView
-var view = new itowns.PlanarView(
-        viewerDiv, extent, { renderer: renderer, maxSubdivisionLevel: 10, camera: camera });
-
-var onMouseWheel = function onMouseWheel(event) {
-    var change = 1 - (Math.sign(event.wheelDelta || -event.detail) * 0.1);
-
-    var halfNewWidth = (view.camera.camera3D.right - view.camera.camera3D.left) * change * 0.5;
-    var halfNewHeight = (view.camera.camera3D.top - view.camera.camera3D.bottom) * change * 0.5;
-    var cx = (view.camera.camera3D.right + view.camera.camera3D.left) * 0.5;
-    var cy = (view.camera.camera3D.top + view.camera.camera3D.bottom) * 0.5;
-
-    view.camera.camera3D.left = cx - halfNewWidth;
-    view.camera.camera3D.right = cx + halfNewWidth;
-    view.camera.camera3D.top = cy + halfNewHeight;
-    view.camera.camera3D.bottom = cy - halfNewHeight;
-
-    view.notifyChange(true);
-};
+var view = new itowns.PlanarView(viewerDiv, extent, { maxSubdivisionLevel: 10 });
+
+// eslint-disable-next-line no-new
+new itowns.PlanarControls(view, {
+    maxAltitude: 40000000,
```

Comment:
Would you mind adding the documentation for these parameters?

Reply:
In PlanarControls' jsdoc

Reply:
Explaining the option values used here would make sense to me too. For example, while reading the example code I would like to understand why `enableRotation` is set to `false`.

Reply:
Doing both !

Reply:
And on the `maxAltitude` value: why not making it bigger? As a user of the example I want to be able to zoom out, and the current `maxAltitude` value prevents it. I tried to change it to 16000000 but the map disappears when I zoom out.

Reply:
Probably another issue with `camera.far`...
#674  would help fixing this by providing a min/max distance for everything we need to display => we could dynamically adjust near/far